deel-3-webpagina/index.html

```diff
@@ -7,10 +7,40 @@
     <title>Flexbox opdracht 3</title>
 </head>
 <body>
-    <div>
-        <h1>All Your Business Data in One Place</h1>
-        <h3>Sync and store all your business data. Explore your data with SQL or visualize it with your favorite BI or
-            analytics tools.</h3>
+<nav class="nav-bar">
+    <p class="company-name">Analytica</p>
+    <ul>
+        <li>About us</li>
+        <li>Strategy</li>
+        <li>Prices</li>
+        <li>Contact</li>
+    </ul>
+</nav>
+
+<div class="opening-title">
+    <h1>All Your Business Data in One Place</h1>
+    <h3>Sync and store all your business data. Explore your data with SQL or visualize it with your favorite BI or
+        analytics tools.</h3>
+</div>
+<main>
+    <div class="inner-container">
+        <article class="lorem-article">
+            <p> Lorem ipsum dolor sit amet, consectetur adipisicing elit. A architecto at beatae deserunt ducimus
```

Comment:
De <p> kan iets netter worden uitgelijnd, dan is het makkelijker om de verschillende elementen te onderscheiden